_pages/lang/developer-engagement.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: page
+title:  "Developer Engagement"
+permalink: /lang/developer-engagement/
```

Comment:
since this is the FR version, please already replace "lang" with "fr" (same for the lang: property below)